src/firefly/js/visualize/task/PlotHipsTask.js

```diff
@@ -188,6 +191,13 @@ export function makePlotHiPSAction(rawAction) {
     };
 }
 
+function addHiPSGridLayer() {
```

Comment:
This confused me a little until I understood what was going on. You might want to call this function `createHiPSGridLayer` since you actually add it somewhere else.